src/cockpit/samples.py

```diff
@@ -387,12 +387,14 @@ def sample(self, samples: Samples) -> None:
                     continue
 
                 path = line.split()[1]
-                res = os.statvfs(path)
-                if res:
-                    frsize = res.f_frsize
-                    total = frsize * res.f_blocks
-                    samples['mount.total'][path] = total
-                    samples['mount.used'][path] = total - frsize * res.f_bfree
+                try:
+                    res = os.statvfs(path)
+                except OSError:
```

Comment:
Ignoring permission errors seems fine to me, but don't we want to know when something really breaks? Although I suppose you can easily see that when you try to `ls` / debug on the machine itself.